Framework/Algorithms/src/MaxEnt/MaxentCalculator.cpp

```diff
@@ -245,7 +258,7 @@ void MaxentCalculator::iterate(const std::vector<double> &data,
       m_coeffs.s1[k][0] += m_directionsIm[k][i] * sgrad[i];
       m_coeffs.c1[k][0] += m_directionsIm[k][i] * cgrad[i];
     }
-    m_coeffs.c1[k][0] /= chiSq;
+    m_coeffs.c1[k][0] /= fac * npoints;
```

Comment:
this will be 4*npoints larger. Why have you changed this?

Reply:
Those probably come from the Fourier transform, I am not sure but after that change the `c1` and `c2` actually become the first and second derivatives of the chi squared at least in the `m_directionsIm[0]` direction.

I just realised that I didn't check the other direction. I'll do that.

Reply:
if it was Fourier transform surely we would expect some multiple of pi?

Reply:
I don't know I just compared the numbers.